libnavigation-core/build.gradle

```
@@ -54,11 +54,13 @@ dependencies {
     implementation project(':libnavigation-util')
     implementation project(':libnavigator')
     runtimeOnly project(':libdirections-hybrid')
-    runtimeOnly project(':libtrip-notification')
+    implementation project(':libtrip-notification')
```

Why do we need to change it?

---

`TripServiceActivity` doesn't compile, because it needs `MapboxTripNotification`

---

By adding `implementation`, we can mistakenly use classes from the implementation package and break modularity.

Would this work?
```suggestion
  runtimeOnly project(':libtrip-notification')
  debugImplementation project(':libtrip-notification')
```

---

Also, if you need the instance of the `MapboxTripNotification`, you can try using the module provider instead.

---

> you can try using the module provider instead

This would be preferable, if possible.

---

fixed